learn/about.md

```diff
@@ -12,12 +12,11 @@ If you're familiar with Rollkit, you may want to skip to the [tutorials section]
 
 Rollkit is a sovereign rollup framework that gives developers the freedom to deploy rollups throughout the modular stack, opening new possibilities for rapid experimentation and innovation.
 
-The Rollkit framework features a modular node that can run rollups and exposes an [ABCI](https://github.com/cometbft/cometbft/tree/main/abci)-compatible client interface, which can be used as a substitute for CometBFT in any ABCI-compatible blockchain application.
```

Comment:
the newly added sentence doesn't cover this information that's deleted. is this intentional?